src/crawlers/browser_crawler.js

```diff
@@ -508,6 +508,11 @@ export default class BrowserCrawler extends BasicCrawler {
             const proxyInfo = this.proxyConfiguration.newProxyInfo(launchContextExtends.session && launchContextExtends.session.id);
             launchContext.proxyUrl = proxyInfo.url;
             launchContextExtends.proxyInfo = proxyInfo;
+
+            if (this.proxyConfiguration.hasRestrictedResidentialProxy) {
+                // @szmarczak: What's the difference with launchContextExtends?
+                launchContext.ignoreHTTPSErrors = true; // Playwright
```

Comment:
```suggestion
                launchContext.ignoreHTTPSErrors = true;
```

this should work on both Playwirght and Puppeteer:

https://playwright.dev/docs/api/class-browser/#browser-new-context
https://github.com/puppeteer/puppeteer/blob/main/docs/api.md

Reply:
looks like, always better to test it yourself :]